src/App.js

```diff
@@ -10,16 +13,6 @@ class App extends React.Component {
     super();
     this.state = {
       todos: [
-        {
-          task: 'Organize Garage',
-          id: 1528817077286,
-          completed: false
-        },
-        {
-          task: 'Bake Cookies',
-          id: 1528817084358,
-          completed: false
-        }
       ],
       todo: ''
     }
```

Comment:
I understand that `todo` here (inside your state) is supposed to hold the value of your input form. However, at first glance, it might not be easy to infer that from the name you gave.

Perhaps you could name this something like `todoInput` or `newTodo` instead to make its purpose clearer.